web/ui/src/features/component/TableHead.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+import { SortOrder } from './types';
+
+interface Props {
+  headers: string[];
+  handleSorting?: (header: string, sortOrder: SortOrder) => void;
+}
+
+const TableHead = ({ headers, handleSorting }: Props) => {
+  const [sortField, setSortField] = useState('');
+  const [order, setOrder] = useState(SortOrder.ASC);
+
+  const handleSortingChange = (header: string) => {
+    const sortOrder = header === sortField && order === SortOrder.ASC ? SortOrder.DESC : SortOrder.ASC;
+    if (handleSorting !== undefined) {
+      setSortField(header);
+      setOrder(sortOrder);
+      handleSorting(header, sortOrder);
+    }
+  };
+
+  return (
+    <tr>
+      {headers.map((header) => {
+        const sortOrderHeaderAttribute = sortField === header ? order : 'default';
+        return (
+          <th
+            key={header}
+            onClick={() => handleSortingChange(header)}
+            data-sort-order={handleSorting ? sortOrderHeaderAttribute : undefined}
```

Comment:
can we change it to:
`data-sort-order={sortField === header ? order : undefined}`
and get rid of `const sortOrderHeaderAttribute = sortField === header ? order : 'default';` ?

Reply:
Done this one too in latest commit.